package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "sbc-auth",
```

Comment:
Please remove this file and update package.json, then update package-lock.json by using `npm i`